test/unit/rules/lint-attribute-indentation-test.js

```diff
@@ -270,13 +475,15 @@ generateRuleTests({
     '{{#t.body' + '\n' +
     '  canExpand=(helper help)' + '\n' +
     '  multiRowExpansion=false' + '\n' +
-    'as |body|}}' + '\n' +
+    'as |body|' + '\n' +
+    '}}' + '\n' +
```

Comment:
I think that this is "fine" (it definitely is correct based on what the rule options are called), but practically speaking this feels _super bad_ to me...

This is a good candidate for some additional follow up after this lands...